recipes/graylog-forwarder/recipe.rb

```diff
@@ -0,0 +1,53 @@
+require_relative '../tools'
+
+class GraylogForwarder < FPM::Cookery::Recipe
+  include Tools
+
+  description 'Graylog Forwarder'
+
+  name     'graylog-forwarder'
+  version  data.version
+  revision data.revision
+  homepage data.homepage
+  arch     'all'
+
+  source data.source
+  sha256 data.sha256
+
+  maintainer data.maintainer
+  vendor     data.vendor
+  license    'see https://www.graylog.org/enterprise'
+
+  targets :deb do
+    section 'net'
+    config_files '/etc/graylog/forwarder/forwarder.conf'
+  end
+
+  targets :rpm do
+    config_files '/etc/graylog/forwarder/forwarder.conf'
+  end
+
+  pre_install    'files/pre-install'
+  post_install   'files/post-install'
+  pre_uninstall  'files/pre-uninstall'
+  post_uninstall 'files/post-uninstall'
+
+  def build
+  end
+
+  def install
+    etc('graylog/forwarder').install file('forwarder.conf'), 'forwarder.conf'
+    share('graylog-forwarder').install 'graylog-cloud-forwarder.jar'
+
+    share('graylog-forwarder').install 'bin'
+    share('graylog-forwarder/bin').install Dir['bin/*']
+
+    share('graylog-forwarder').install 'config'
+    share('graylog-forwarder/config').install Dir['config/jvm.options']
```

Comment:
This file should go do `/etc/graylog/forwarder/jvm.options` because it might be changed by users. We also need to add `/etc/graylog/forwarder/jvm.options` to `config_files` to make sure we don't overwrite that file when users have modified it.

Reply:
I can move the file. We'll have to change the calling script, too, though.

https://github.com/Graylog2/graylog-plugin-cloud/blob/cloud-3.3/forwarder/bin/graylog-forwarder.sh#L34

Reply:
Changing the file is okay. We can either create a patch for the file like we do [here](https://github.com/Graylog2/fpm-recipes/blob/master/recipes/graylog-server/patches/graylog-server.conf.patch) or we adjust the script in the cloud repository to make it possible to override the variable via environment variable. (which can be done in the systemd service file)

```diff
- FORWARDER_JVM_OPTIONS_FILE="$FORWARDER_ROOT/config/jvm.options"
+ FORWARDER_JVM_OPTIONS_FILE="${FORWARDER_JVM_OPTIONS_FILE:-$FORWARDER_ROOT/config/jvm.options}"
```

Reply:
Sounds good. :+1:

Reply:
This is fixed.